src/_posts/aframe-v0.3.0.md

```diff
@@ -9,9 +9,9 @@ image:
 
 [webvr1.0]: https://w3c.github.io/webvr/
 
-A-Frame **v0.3.0** has dropped! With support for the **[new WebVR 1.0
+A-Frame **version 0.3.0** has dropped! With support for the **[new WebVR 1.0
 API][webvr1.0]**, we aim higher towards state-of-the-art VR. With room-scale
-tracking, we can walk in a web page, and with tracked controller support, we
+tracking, we can walk in a web page; and with tracked controller support, we
```

Comment:
semicolons can be used in two cases: (1) to punctuate two independent clauses and (2) to separate items in lists where commas are being used (e.g., `We have locations in City, State; City, State; and City, State.`)